### PR TITLE
Initial code housekeeping

### PR DESCRIPTION
**Pull request by jeffreytjs**
Fixed typos and formatting.
Remove unused codes.

Does not cover Tests

### CODE REVIEW

File: src/main/java/seedu/clinic/model/product/Product.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import seedu.clinic.model.attribute.Tag;
 
 /**
- * Represents Product sold by Suppliers or stored in Warehouse.
+ * Represents Product sold by a Suppliers or stored in Warehouse.
```
Comment (zhengweii) [inline src/main/java/seedu/clinic/model/product/Product.java]: Minor typo here! Should be "Supplier"!

**Review by zhengweii — APPROVED** [review]
Apart from the typo, everything looks great as well! Thanks for helping to clean up the code!

File: src/main/java/seedu/clinic/model/product/Product.java (hunk 1 above)
Comment (jeffreytjs) [reply src/main/java/seedu/clinic/model/product/Product.java]: Opps it wasn't meant to be changed but I forgot to change it back

**Review by Criss-Wang — COMMENTED** [review]
Really detailed work here! Thanks Jeffrey! I'm thinking this can be the last one PR to merge as it cleans up our code entirely!

**Pull request merged**